lsp-mode.el

```diff
@@ -5853,12 +5844,11 @@ perform the request synchronously."
                        (read-string (format "Rename %s to: " symbol) placeholder nil symbol))))
   (unless newname
     (user-error "A rename is not valid at this position"))
-  (let ((edits (lsp-request "textDocument/rename"
-                            `(:textDocument ,(lsp--text-document-identifier)
-                                            :position ,(lsp--cur-position)
-                                            :newName ,newname))))
-    (when edits
-      (lsp--apply-workspace-edit edits))))
+  (when-let ((edits (lsp-request "textDocument/rename"
+                                 `(:textDocument ,(lsp--text-document-identifier)
+                                                 :position ,(lsp--cur-position)
+                                                 :newName ,newname))))
```

Comment:
these lines should be reverted

Reply:
Would you mind explaining? I can't spot any obviously-important difference in the expanded forms.

Reply:
Sorry I didn't notice the before change diff.
These should be change so the `:position` is aligned with `:textDocument` and so on